### ISSUE TITLE
Entry is created with wrong password

### ISSUE THREAD

**Issue by Pasukaru** [1Password/terraform-provider-onepassword#139]
### Your environment

Terraform Provider Version: 1.4.0

CLI Version: 2.23.0

OS: linux (CircleCI), MacOS (local)

Terraform Version: 1.6.6

## What happened?
Terraform apply initially creates the item with a wrong password being set.
Running terraform apply (without any changes to the tf config) then updates the item with the correct password.

## What did you expect to happen?
Terraform apply should create the item with the correct password on first execution.

## Steps to reproduce
```
resource "onepassword_item" "basic_auth" {
  vault    = local.onePasswordVault
  tags     = local.onePasswordTags
  title    = "${local.onePasswordTitlePrefix}Basic Auth"
  category = "login"
  username = local.basicAuthUsername
  password = local.basicAuthPassword
  url      = "https://${var.domain}"
}
```

## Notes & Logs
I noticed this because the login wasn't working, and upon checking the password in the the 1pw entry, it was in a scheme that didn't match our password generation in terraform.

Possibly related to high concurrency. When a new environment is created, many different 1password entries are generated.
I have not encountered this in a case where terraform only needs to update/create a single item.

The second terraform apply logged this:
```

Terraform will perform the following actions:

  # onepassword_item.basic_auth will be updated in-place

 ~ resource "onepassword_item" "basic_auth" {
        id       = "vaults/redacted/items/redacted"
      ~ password = (sensitive value)
        tags     = [
            "terraform",
        ]
        # (6 unchanged attributes hidden)
    }
```

I don't have logs of the first run anymore, will capture them when it happens again. But from what I recall, nothing special there, just the regular `Terraform will create this resource`.

**Comment by simonweil** [comment]
I have this issue too, very annoying...

**Comment by volodymyrZotov** [comment]
@Pasukaru Thank you for raising this! We'll take a look into that and make a release with the fix!

**Comment by volodymyrZotov** [comment]
Fixed in [v1.4.1](https://github.com/1Password/terraform-provider-onepassword/releases/tag/v1.4.1)
